commands/realm_client.go

```diff
@@ -0,0 +1,33 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+	"fmt"
+)
+
+func init() {
+	realmCmd.AddCommand(realmClientCmd)
+}
+
+var realmClientCmd = &cobra.Command{
+	Use:   "client [realm clientId]",
+	Short: "Get information about a client",
+	Long:  "Get information about a client",
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		realm := args[0]
```

Comment:
We already have a top level "keycloak-realm" so my question is, do we need to pass this in here. (At least I dont think)

```
--keycloak-realm string      Realm to use (default "telia")
```

Reply:
```
$ go run cmd/cli/main.go realm client telia 6086e06a017c4651
ClientId 6086e06a017c4651 has name New self service and description 
```

```
Chriss-MacBook-Pro:divx-taas-keycloak-golang freshteapot$ go run cmd/cli/main.go --keycloak-realm bob realm client telia 6086e06a017c4651
2018/05/23 11:29:30 Failed to get an oidc Provider
exit status 1
```

Reply:
I have just tried to mimic what was already done, so if you think we don't need the realm, we go for your solution :)

Reply:
I will fix it up.